docs/mkdocs-nedoc-plugin/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
+    name="mkdocs-nedoc-plugin",
+    version="0.0.1",
+    description="Nedoc plugin for linking to API documentation",
+    author="Stanislav Böhm",
+    author_email="spirali@kreatrix.org",
```

Comment:
The author should be you :)

Reply:
It's a local private package anyway (at least for now). We can change it later if we need it in multiple projects and we put it on PyPi :)